llm-eval/feedback.py

```diff
@@ -4,8 +4,8 @@
 import numpy as np
 import os
 
-# replicate key for running feedback
-os.environ["REPLICATE_API_TOKEN"] = st.secrets["REPLICATE_API_TOKEN"]
+from dotenv import load_dotenv
```

Comment:
nit: we should probably keep all keys in one place. I ended up using streamlit secrets instead of dotenv so that  another dependency isn't needed